File: sandbox/node/src/FaxLineAddUserDefaultExample.ts
import * as fs from 'fs';
import api from "@dropbox/sign"
import models from "@dropbox/sign"

const apiCaller = new api.FaxLineApi();

const faxLineAddUserRequest = new models.FaxLineAddUserRequest();
faxLineAddUserRequest.number = "[FAX_NUMBER]";
faxLineAddUserRequest.emailAddress = "member@dropboxsign.com";

apiCaller.faxLineAddUser(
    faxLineAddUserRequest,
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling FaxLine#faxLineAddUser:");
  console.log(error.body);
});

File: sandbox/dotnet/src/Dropbox.SignSandbox/FaxLineAddUserDefaultExample.cs
using System;
using System.Collections.Generic;
using System.IO;

using Dropbox.Sign.Api;
using Dropbox.Sign.Client;
using Dropbox.Sign.Model;

namespace Dropbox.SignSandbox;

public class FaxLineAddUserDefaultExample
{
    public static void Run()
    {
        var config = new Configuration();

        var faxLineAddUserRequest = new FaxLineAddUserRequest(
            number: "[FAX_NUMBER]",
            emailAddress: "member@dropboxsign.com"
        );

        try
        {
            var response = new FaxLineApi(config).FaxLineAddUser(
                faxLineAddUserRequest: faxLineAddUserRequest
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling FaxLine#FaxLineAddUser: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}
